frb/scripts/frb_summary.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""
+Script to print a summary of a given FRB to the terminal
+"""
+from IPython import embed
+
+def parser(options=None):
+    import argparse
+    # Parse
+    parser = argparse.ArgumentParser(description='Script to print a summary of an FRB to the screen [v1.0]')
+    parser.add_argument("frb_name", type=str, help="FRB name, e.g. FRB180924 or simply 180924")
+    parser.add_argument("--verbose", default=False, action="store_true", help="Overwhelm the screen?")
+
+    if options is None:
+        pargs = parser.parse_args()
+    else:
+        pargs = parser.parse_args(options)
+    return pargs
+
+
+def main(pargs):
+    """ Run
+    """
+    import warnings
+    import numpy as np
+    import json
+
+    from linetools import utils as ltu
+
+    from frb import frb
+
+    # Load
+    frb_name = pargs.frb_name if pargs.frb_name[0] == 'F' else 'FRB'+pargs.frb_name
```

Comment:
`pargs.frb_name[:3] == 'FRB'` would be a stronger check. Also, if you want to allow the user to be even sloppier, I'd implement:
```python
frb_name = pargs.frb_name if pargs.frb_name[:3].upper() == 'FRB' else 'FRB'+pargs.frb_name[3:]
```
This way, inputs like `frb180924` can also be accepted.